201-api-management-logs-to-moesif-using-eventhub-webapp/azuredeploy.json

```diff
@@ -252,7 +254,35 @@
                     }
                 }
             }
-        }
+        }/* If this section 'app-service-webjob-msdeploy' is uncommented, 
```

Comment:
not sure I follow - is this needed or not?  Seems like not (since it's commented out) we should just remove it.

Reply:
@bmoore-msft 

You had asked me on Jan 6 [Link](https://github.com/Azure/azure-quickstart-templates/pull/8638#discussion_r552779802):
```
bmoore-msft on Jan 6
That seems fine - but for now remove it if we're not supporting MSDeploy in this sample and we can publish. 
Then you can submit another PR with MSDeploy 
and we can see if we can debug what's happening there.
```
Now, I am submitting the new PR. I would like to enable deployment of `app-service-webjob-msdeploy` template. This template deploys a `webjob` on azure app service using `msdeploy`. 
The only reason why I disabled this template for deployment is because I cannot request review from you .
When I enable this template, it exposes a potential bug in Azure ARM deployment (I believe .. in my humble opinion).  
The bug is that: despite the fact `app-service-webjob-msdeploy` dependsOn `app-service-deploy` the ARM deployment fails. But despite the failure, if we dont do anything, after 5 minutes, the webjob starts running.
I think issue might be that DNS propagation of newly deployed site required for MSDeploy to succeed does not happen within the expected timeframe for Azure deployment.

Reply:
If we want a sample with MSDeploy, uncomment the code and submit the sample so we can see the code and the error.